src/Illuminate/Support/Testing/Fakes/HashFake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Illuminate\Support\Testing\Fakes;
+
+use Illuminate\Contracts\Hashing\Hasher;
+
+class HashFake implements Hasher
+{
+    /**
+     * Get information about the given hashed value.
+     *
+     * @param  string  $hashedValue
+     * @return array
+     */
+    public function info($hashedValue)
+    {
+        return password_get_info($hashedValue);
+    }
+
+    /**
+     * Hash the given value.
+     *
+     * @param  string  $value
+     * @param  array   $options
+     * @return string
+     */
+    public function make($value, array $options = [])
+    {
+        return crc32($value);
```

Comment:
So you should cast this to a string, or better:
* `sprintf('%08s', dechex(crc32($str)))`
* `hash('crc32b', $str)`